lib/src/event.dart

```diff
@@ -1019,10 +1017,7 @@ class Event extends MatrixEvent {
   /// This is useful to determine if stand-alone emotes should be displayed bigger.
   bool get onlyEmotes {
     if (isRichMessage) {
-      final formattedTextStripped = formattedText.replaceAll(
-          RegExp('<mx-reply>.*</mx-reply>',
-              caseSensitive: false, multiLine: false, dotAll: true),
-          '');
+      final formattedTextStripped = calcUnlocalizedBody(hideReply: true);
```

Comment:
Can you explain, what exactly this is fixing? Because I think it might not actually remove the reply fallback now?

Reply:
Actually, no, it seems to strip the reply fallback, the html to text convert does that, it just isn't obvious....

Reply:
https://github.com/famedly/matrix-dart-sdk/pull/1924 is somewhat related, where I just try to figure out what that function actually does :p

Reply:
Here we are, I added a comment string on why this is failing right now.